stko/calculators/macromodel_calculators.py

```diff
@@ -0,0 +1,250 @@
+"""
+MacorModel Calculators
+=================
+
+# . :class:`.MacroModelForceFieldEnergy`
+
+Wrappers for calculators that use Schrödinger's MacroModel software.
+
+"""
+
+import logging
+from .calculators import Calculator
+from ..optimizers import MacroModel
```

Comment:
I do not love that this inherits from an optimizer. Maybe the `MacroModel` class you have here should be a `Wrapper` (containing software specific things), then the optimizer and energy classes inherit/use (maybe use is better!) those?